app/main/controller/image_similarity_controller.py

```diff
@@ -21,8 +21,12 @@ class ImageSimilarityResource(Resource):
   def post(self):
     result = True
     image = ImageModel.from_url(request.json['url'], request.json['context'])
-    db.session.add(image)
-    db.session.commit()
+    try:
```

Comment:
are you sure this still logs the exception? doesn't `try` absorb it?

Reply:
that's why there is a `raise` in the `except` block

Reply:
ohhhh I didn't notice it